migrate/src/main/scala/migrate/interfaces/MigrateImpl.scala

```diff
@@ -0,0 +1,8 @@
+package migrate.interfaces
+
+import migrate.interfaces.Migrate
+import migrate.interfaces.MigrateService
+
+final class MigrateImpl extends Migrate {
```

Comment:
I don't think we need 2 interfaces `Migrate` and `MigrateService` since none of them takes parameters at constructution.

Reply:
ok.

Reply:
I like to have a clean interface, where we don't see the method that classload.. but Right now, I only have one method. So I will remove it and see how it grows